helm/README.md

```diff
@@ -101,6 +101,13 @@ To enable any of the products to use HTTPS on their external endpoint, set the f
 useTLS: True
 ```
 
+The default behaviour if useTLS = true, is to use certmanager to manage the certificate request/renewal via letsencrypt.  This is enabled by the following flag:
```

Comment:
Sugg: Put this comment above useTLS: true:

The default behavior is to use TLS and have certmanager to manage the certificate request/renewal via 
letsencrypt.

If you want to use TLS but don't want to certmanager to manage the certificate request/renewal, then set
useTLS to true, set useCertManager to false, and run the script ../bin/generate-tls.sh.

Reply:
I changed the 2nd section as suggested but as useTLS isn't enabled by default as part of this PR, I can't change the first section quite yet.  I'll be changing this tomorrow once I've coordinated with Pavel as it impacts his tests.